doc/man1/openssl-dgst.pod

```diff
@@ -52,7 +52,7 @@ command.
 
 Print out a usage message.
 
-=item B<-I<digest>>
+=item B<-> I<digest>
```

Comment:
And this shows why this change is a bad idea. That space is *plain wrong*

Reply:
space removed.